Dolapo47/Auto-Mart
# create protection for routes and refactor test file

#### What does this PR do?
- A summary of what this task is about
this PR implements protection for routes and also implements test cases for the user routes
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
create a token generator and token verifier middle-ware
create test cases for the user routes
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
you can test the route using 'npm test'
you can view the user routes using 'npm start'
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166711931](https://www.pivotaltracker.com/n/projects/2346179)`)
